ansible/roles/proteus-db/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Create database schema files
+  copy:
+    src: "{{ item }}"
+    dest: "{{ proteus_db_schema_path }}"
+    mode: 0644
+  with_items: "{{ proteus_db_files }}"
+
+- name: Run database schema commands
+  shell: "psql -U {{ proteus_db_user}} -h 127.0.0.1 -d {{ proteus_db_name }} -f {{ proteus_db_schema_path}}/{{ item }}"
+  environment:
+    PGPASSWORD: "{{ proteus_db_password }}"
+  with_items: "{{ proteus_db_files }}"
```

Comment:
Actually all of this stuff can probably be removed as it's now part of proteus itself. We only need to create the proteus_db_user and proteus will take care of populating the database.